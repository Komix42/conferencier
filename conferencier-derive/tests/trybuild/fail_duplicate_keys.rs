#[derive(conferencier_derive::ConferModule)]
struct DuplicateKeys {
    one: i32,
    #[confer(rename = "one")]
    two: i32,
}

fn main() {}
