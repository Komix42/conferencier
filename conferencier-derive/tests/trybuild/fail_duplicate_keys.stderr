error: duplicate TOML key `one` detected (previously declared here)
 --> tests/trybuild/fail_duplicate_keys.rs:4:5
  |
4 |     #[confer(rename = "one")]
  |     ^

error: see previous definition
 --> tests/trybuild/fail_duplicate_keys.rs:3:5
  |
3 |     one: i32,
  |     ^^^
